playwright.config.ts

```diff
@@ -20,6 +13,8 @@ export default defineConfig({
   globalTimeout: minutesToMilliseconds(60),
   /* Run tests in files in parallel */
   fullyParallel: false,
+  /* Ensure tests run consecutively due to inability to share wiremock instance */
```

Comment:
noticed the tests were a bit flaky and it's because multiple workers were messing around with wiremock at the same time

Reply:
yes I saw they were flaky locally but seemed to always work in CircleCI, I wasn't sure why

Reply:
I think it must try and work out the workers dynamically based on cpu - circle maybe low spec so it defaults to one? 🤷